doc/requirements.txt

```diff
@@ -1,2 +1,2 @@
-http://athyra.ged.msu.edu/~mcrusoe/autoprogram/sphinxcontrib-autoprogram-0.1.1-khmerdev-20140331.tar.gz#egg=sphinxcontrib-autoprogram==0.1.1-khmerdev-20140331
+http://ci.oxli.org:8080/userContent/sphinxcontrib-autoprogram-0.1.1-khmerdev-20140331.tar.gz#egg=sphinxcontrib-autoprogram==0.1.1-khmerdev-20140331
```

Comment:
See the entry for that commit (the port number is needed when downloading from ci.oxli.org itself) https://github.com/dib-lab/khmer/commit/960257b75751dd8f1e9037b3bac6e5b12ebb1601